.maestro/shared/delay.yaml

```diff
@@ -0,0 +1,12 @@
+# delay.yaml
+
+appId: com.duckduckgo.mobile.ios
+---
+
+- runFlow:
+    when:
+      visible:
+        text: "This is just a text that will be never matched so this invocation will cause 5s delay"
+        index: 0
+    file: onboarding_browsing.yaml
```

Comment:
Anything is fine here obviously. I'm just wondering if something like
```
commands: 
  - stopApp: appId
```

would make intentions clearer here.

Reply:
Thanks, I've tweaked the command